app/controllers/api/v1/topics_controller.rb

```diff
@@ -0,0 +1,81 @@
+module Api
+  module V1
+    class TopicsController < ApplicationController
+
+      # before_action :set_topic, only: %i[show edit update destroy]
+
+      # GET /topics or /topics.json
+      def index
+        @topics = Topic.all
+      end
+
+=begin
+      # GET /topics/1 or /topics/1.json
+      def show
+      end
+
+      # GET /topics/new
```

Comment:
Please avoid comments

Reply:
Okay, I used the other methods because I created topics with that methods, but I found how use seeds to create the initial topics, I going to remove the unnecessary methods.
Thanks for suggestion.